AmethystAPI/src/minecraft/src-client/common/client/renderer/game/LevelRendererPlayer.cpp

```diff
@@ -0,0 +1,6 @@
+#include "minecraft/src-client/common/client/renderer/game/LevelRendererPlayer.h"
+
+float LevelRendererPlayer::getFov(float originalFov, bool applyEffects) {
+    static auto func = reinterpret_cast<LevelRendererPlayer::_getFov>(SigScan("48 8B C4 48 89 58 ? 48 89 70 ? 57 48 81 EC ? ? ? ? 0F 29 70 ? 0F 29 78 ? 44 0F 29 40 ? 44 0F 29 48 ? 48 8B 05"));
```

Comment:
same feedback for not using the _typedefs. For calling static functions, I think it makes more sense to do this consistently in the project

```
using Function = float(__thiscall*)(LevelRendererPlayer*, float, bool);
static auto func = reinterpret_cast<Function>(SigScan(...));
return func(this, ...)
```